lib/commands/app/rename.ts

```diff
@@ -39,16 +39,19 @@ export default class AppRenameCmd extends Command {
 
 		Note, if the \`newName\` parameter is omitted, it will be
 		prompted for interactively.
-		`;
+	`;
+
 	public static examples = [
 		'$ balena app rename OldName',
 		'$ balena app rename OldName NewName',
+		'$ balena app rename myorg/oldname NewName',
 	];
 
 	public static args: Array<IArg<any>> = [
 		{
-			name: 'name',
-			description: 'application name or numeric ID',
+			name: 'nameOrSlug',
+			description: 'application name or org/name slug',
+			parse: lowercaseIfSlug,
```

Comment:
Converting input to lowercase if slug (on Akis' suggestion), since slugs are lowercase.